test/e2e/config/caren.yaml

```diff
@@ -134,8 +134,8 @@ providers:
 - name: caren
   type: RuntimeExtensionProvider
   versions:
-  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.14}"
-    value: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/download/{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.14}/runtime-extension-components.yaml"
+  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.15}"
```

Comment:
@jimmidyson can you please verify if the main branch should be using current minor release from `main` ?

Reply:
Until we get the upgrade test in place this is actually unused so I'm happy to merge this as it is and we can revisit this when we get the upgrade test in place. I think we might want to have both previous patch and previous minor in here and use both in upgrade tests but like I said let's discuss that when we come to it.

Reply:
thank you for the details..